CIVians/chains.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "CIVians",
+  "chains": [
+    {
+      "name": "unification",
+      "address": "undvaloper10rrwge9erexjwqs52k4cagz6m8x68jutua4e64",
+      "restake": {
+        "address": "und1qz65fr7dp2vle7el76lhdxhxwa3rs2vh4fezvs",
+        "run_time": "22:22 UTC",
```

```suggestion
        "run_time": "22:22",
```
Times are required to be in UTC and this could cause issues when it's parsed